network.py

```diff
@@ -76,15 +76,17 @@ def __init__(self, address, port):
 
     def receive(self):
         """
-        Receives data and tries to parse it into an integer containing
-        the time of the server player in milliseconds.
+        Receives data and tries to parse it into a list of integers containing
+        the current playlist position, and time of the server player in milliseconds.
         """
         try:
             data = self.sock.recv(4096)
             if data:
                 data = data.decode()
-                pos = data.split(',')[-2]
-                return int(pos)
+                # playlist_position,time
+                server_state_strings = [data.split(',')[-3], data.split(',')[-2]]
```

Comment:
Probably negligible tbh, but I wonder if/how much faster assigning the split would be first?
```
data_strings = data.decode().split(',')
server_state_strings = data_strings(data_strings[-3], data_strings[-2]
```

Reply:
@sam-maher-acmi Nice one. I tested the speed and it's around 0.001 milliseconds difference, but I'll clean it up anyway.